<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataView</class>
 <widget class="QDialog" name="DataView">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>649</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Signal space analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>SystemLabIcon.bmp</normaloff>SystemLabIcon.bmp</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabData">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_xy">
      <attribute name="title">
       <string>Signal space (I-Q)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <item row="0" column="1">
        <widget class="QFrame" name="graphFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="dataFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>159</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QLineEdit" name="totalSamplesTime">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>76</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sampling pts:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="evm_per">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>28</x>
            <y>30</y>
            <width>58</width>
            <height>16</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>EVM (%):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="evm_db">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>27</x>
            <y>48</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>EVM (dB):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>156</y>
            <width>141</width>
            <height>71</height>
           </rect>
          </property>
          <property name="title">
           <string>Signal type (for viewing)</string>
          </property>
          <widget class="QCheckBox" name="signalCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>24</y>
             <width>131</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Symbol (reference)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="sigandnoiseCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>44</y>
             <width>131</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Symbol (received)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>74</y>
            <width>141</width>
            <height>76</height>
           </rect>
          </property>
          <property name="title">
           <string>Iterations</string>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>46</x>
             <y>30</y>
             <width>46</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Current:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBoxTime">
           <property name="geometry">
            <rect>
             <x>94</x>
             <y>26</y>
             <width>41</width>
             <height>23</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
          <widget class="QLineEdit" name="totalIterationsTime">
           <property name="geometry">
            <rect>
             <x>94</x>
             <y>54</y>
             <width>31</width>
             <height>17</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>4</x>
             <y>54</y>
             <width>85</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Total iterations:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>232</y>
            <width>141</width>
            <height>70</height>
           </rect>
          </property>
          <property name="title">
           <string>Grid lines</string>
          </property>
          <widget class="QCheckBox" name="checkBoxMajorGrid">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>26</y>
             <width>119</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Display major grid</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBoxMinorGrid">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>44</y>
             <width>118</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Display minor grid</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
